core/zipline/src/jsMain/kotlin/io/github/droidkaigi/confsched2022/zipline/JsScheduleModifier.kt

```diff
@@ -38,6 +40,17 @@ private fun TimetableItem.modified(): TimetableItem {
         copy(
             targetAudience = "To be written.",
         )
+    } else if (
+        this is Session &&
```

Comment:
Sorry, we probably can't use the description or the message. Because it is Special Session.
But we can use it as a sample. So let me merge it 🙏